docs/source/workbench/sliceviewer.rst

```diff
@@ -0,0 +1,137 @@
+.. _sliceviewer:
+
+===========
+Sliceviewer
+===========
+
+.. contents:: Table of Contents
+   :local:
+
+Overview
+--------
+
+.. figure:: ../images/wb-sliceviewer51-roi.png
+   :class: screenshot
+   :align: center
+
+The Sliceviewer is a viewer supporting :ref:`MDEventWorkspace <MDWorkspace>`, :ref:`MDHistoWorkspace <MDWorkspace>` & :ref:`MatrixWorkspace`.
+For multi-dimensional workspaces, you can view both MDWorkspaces and MDHistoWorkspaces, a 2D "slice" of the higher dimensional space
+is shown, whereas a MarixWorkspace simply shows the 2 defined dimensions.
+
+Some of the key features of the Sliceviewer provided are:
+
+- Overlay of peaks workspaces
+- Non-orthogonal axes view mode, including peaks workspace overlays
+- ROI preview and extraction
+- Cursor information widget that, for a MatrixWorkspace, includes quantities such as :math:`l1, l2, 2\theta` etc
+- Arrow keys can now be used to move the cursor a pixel at a time when single-pixel line plots are enabled
+
+The following sections illustrate some of these key features.
+
+.. _sliceviewer_peaks_overlay:
+
+PeaksWorkspace Overlay
+----------------------
+
+The peaks overlay button allows selection of one or more PeaksWorkspaces to display on top of the main data image.
+This is only enabled for MD workspaces.
+
+.. figure:: ../images/wb-sliceviewer51-peaksbutton.png
+   :class: screenshot
+   :align: center
+
+Peaks are displayed at the locations defined by each peak center
+with an 'x', while optionally displaying any peak shape if a given peak has
+been integrated.
+
+.. figure:: ../images/wb-sliceviewer51-peaksoverlay.png
+   :class: screenshot
+   :width: 75%
+   :align: center
+
+
+Non-Orthogonal Axes View
+------------------------
+
+If the units of an MDWorkspace are HKL and the workspace has an attached
+UB matrix then the option to display the image using axes aligned with the
+crystal (so-called non-orthogonal view) will be now be visible:
```

Comment:
Such a minor nit-pick, but I don't think this is technically true. If a workspace fulfils your criteria (i.e. has HKL units and a UB present)  but the angles are set to 90 (within some tolerance?) then the button is not enabled (as you would expect!). However I believe the meaning is clear and I think this should PR should be merged before smoke testing so I am approving (I'm just sorry I didn't pick up on it before).